### PR TITLE
Add ClusterName and Region to response

### PR DESCRIPTION
**Pull request by Smirl**
# Summary

- adding ClusterName and Region to the proto file and code
- allowing multiple environment variables to be used to get this kubernetes info
- add tests for the above changes to `GetEnv`
- allow viper config to be set via Env vars also
  - currently we can only set viper config via flags and a config file.
  - it is sometimes easier to use environment variables
  - nested variables (like `grpc.port`) can be specified with `GRPC_PORT` variable
  - There is no prefix set for the environment variables.
- fix golangci-lint issues
  - fix grpc warnings https://pkg.go.dev/google.golang.org/grpc#WithInsecure
  - use `rr.Body.Bytes()` instead of `[]byte(rr.Body.String())`
-pin `arduino/setup-protoc@v1`

### CODE REVIEW

File: internal/helpers/env.go [hunk 1]
```diff
@@ -2,11 +2,14 @@ package helpers
 
 import "os"
 
-// GetEnv retrieves an environment variable with a default value
+// GetEnv retrieves list of possible environment variables with a default value
 // if the environment variable is missing
-func GetEnv(key, fallback string) string {
-	if value, ok := os.LookupEnv(key); ok {
-		return value
+func GetEnv(keys_and_fallback ...string) string {
```
Comment (maruina) [inline internal/helpers/env.go:7]:
```suggestion
func GetEnv(keysAndFallback ...string) string {
```
How do you feel about using camelCase here?

File: pkg/infrabin/grpc_test.go [hunk 2]
```diff
@@ -39,7 +40,7 @@ func TestShutdown(t *testing.T) {
 		context.Background(),
 		"bufnet",
 		grpc.WithContextDialer(func(ctx context.Context, address string) (net.Conn, error) { return lis.Dial() }),
-		grpc.WithInsecure(),
+		grpc.WithTransportCredentials(insecure.NewCredentials()),
```
Comment (maruina) [inline pkg/infrabin/grpc_test.go:43]: Can you add a note about this change?

File: pkg/infrabin/config.go [hunk 3]
```diff
@@ -38,6 +39,8 @@ func ReadConfiguration() {
 	// Config file should be config.yaml
 	viper.SetConfigName(DefaultConfigName)
 	viper.SetConfigType(DefaultConfigType)
+	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
+	viper.AutomaticEnv()
```
Comment (maruina) [inline pkg/infrabin/config.go:43]: Can you add a note about why this change?

**Review by maruina — COMMENTED** [review]
LGTM, just a couple of comments before approving it

File: pkg/infrabin/config.go (hunk 3 above)
Comment (Smirl) [reply pkg/infrabin/config.go:43]: left a note in the PR description

**Review by maruina — APPROVED** [review]
